_inc/client/traffic/verification-services.jsx

```diff
@@ -16,11 +16,30 @@ import {
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
+import JetpackBanner from 'components/jetpack-banner';
 
 export const VerificationServices = moduleSettingsForm(
 	React.createClass( {
+		activateVerificationTools: function() {
+			return this.props.updateOptions( { [ 'verification-tools' ]: true } );
+		},
+
 		render() {
 			const verification = this.props.getModule( 'verification-tools' );
+
+			// Show one-way activation banner if not active
+			if ( ! this.props.getOptionValue( 'verification-tools' ) ) {
+				return (
+					<JetpackBanner
+						callToAction={ __( 'Activate' ) }
+						title={ verification.name }
+						icon="cog"
+						description={ verification.long_description }
+						onClick={ this.activateVerificationTools }
+					/>
```

Comment:
<img width="599" alt="captura de pantalla 2017-08-08 a las 14 43 17" src="https://user-images.githubusercontent.com/1041600/29086212-34a67052-7c48-11e7-8648-3db80de82eef.png">

There's something odd with this, the svg is included twice.
This is not happening with other banners like the support banner at the bottom.